sdk/communication/communication-chat/src/models/models.ts

```diff
@@ -1,86 +1,95 @@
-   */
-  sender?: CommunicationUserIdentifier;
-  /**
-   * Content of a chat message.
-   */
+/** Chat thread. */
+export interface ChatThread {
+  /** Chat thread id. */
+  id: string;
+  /** Chat thread topic. */
+  topic: string;
+  /** The timestamp when the chat thread was created. The timestamp is in RFC3339 format: `yyyy-MM-ddTHH:mm:ssZ`. */
+  createdOn: Date;
+  /** Identifies a participant in Azure Communication services. A participant is, for example, a phone number or an Azure communication user. This model must be interpreted as a union: Apart from rawId, at most one further property may be set. */
+  readonly createdBy?: CommunicationIdentifier;
+  /** The timestamp when the chat thread was deleted. The timestamp is in RFC3339 format: `yyyy-MM-ddTHH:mm:ssZ`. */
+  deletedOn?: Date;
+}
+
+/** Chat message. */
+export interface ChatMessage {
+  /** The id of the chat message. This id is server generated. */
+  id: string;
+  /** The chat message type. */
+  type: ChatMessageType;
+  /** Sequence of the chat message in the conversation. */
+  sequenceId: string;
+  /** Version of the chat message. */
+  version: string;
+  /** Content of a chat message. */
   content?: ChatMessageContent;
+  /** The display name of the chat message sender. This property is used to populate sender name for push notifications. */
+  senderDisplayName?: string;
+  /** The timestamp when the chat message arrived at the server. The timestamp is in RFC3339 format: `yyyy-MM-ddTHH:mm:ssZ`. */
+  createdOn: Date;
+  /** Identifies a participant in Azure Communication services. A participant is, for example, a phone number or an Azure communication user. This model must be interpreted as a union: Apart from rawId, at most one further property may be set. */
+  sender?: CommunicationIdentifier;
+  /** The timestamp (if applicable) when the message was deleted. The timestamp is in RFC3339 format: `yyyy-MM-ddTHH:mm:ssZ`. */
+  deletedOn?: Date;
+  /** The last timestamp (if applicable) when the message was edited. The timestamp is in RFC3339 format: `yyyy-MM-ddTHH:mm:ssZ`. */
+  editedOn?: Date;
 }
 
-export interface ChatMessageContent extends Omit<RestChatMessageContent, "participants"> {
-  /**
-   * Chat message content for type "participantAdded" or "participantRemoved" messages.
-   */
+/** Content of a chat message. */
+export interface ChatMessageContent {
+  /** Chat message content for messages of types text or html. */
+  message?: string;
+  /** Chat message content for messages of type topicUpdated. */
+  topic?: string;
+  /** Chat message content for messages of types participantAdded or participantRemoved. */
   participants?: ChatParticipant[];
+  /** Identifies a participant in Azure Communication services. A participant is, for example, a phone number or an Azure communication user. This model must be interpreted as a union: Apart from rawId, at most one further property may be set. */
+  initiator?: CommunicationIdentifier;
```

Comment:
```suggestion
  initiator?: CommunicationIdentifierKind;
```

Reply:
fixed

Reply:
CommunicationIdentifierKind is different than in the other sdks. Was that the plan with JS @DominikMe ?